docs/docs/overview/why_secretless.md

```diff
@@ -6,32 +6,36 @@ description: Secretless Broker Documentation
 permalink: docs/overview/why_secretless.html
 ---
 
-The Secretless Broker is designed to solve two problems. The first is **loss or theft of credentials from applications and services**, which can occur by:
+The Secretless Broker is primarily designed to solve two problems. The first is
+**loss or theft of credentials from applications and services**, which can occur by:
 
 - Accidental credential leakage (e.g. credential checked into source control, etc)
 - An attack on a privileged user (e.g. phishing, developer machine compromises, etc)
 - A vulnerability in an application (e.g. remote code execution, environment variable dump, etc)
 
-The second is **downtime caused when applications or services do not respond to credential rotation** and crash or get locked out of target services as a result.
+The second is the **changes to code that interacting with a secrets vault requires,
+and the lack of flexibility that allows**.
```

Comment:
I think this sentence needs just a bit of clarity change - took me a couple of re-reads to understand what it was trying to say.

Reply:
you're not wrong. any suggestions?

Reply:
"The second is the need for a client app to know how to interact with secrets vaults" maybe?